### DISCUSSION TITLE
MakeFile installation, make command problems

### DISCUSSION THREAD

**Discussion by ZanyBonzy** [Cyfrin/foundry-full-course-cu#235] [General]
HI everyone.  
Can't install MakeFile on windows pc.
1. Tried the vscode MakeFile extension (successfully installed), but make command not found upon running;
2. Tried choco, "error occured during installation", didn't install;
3. Tried Mingw, "file no downloaded correctly??"

Every idea, suggestion will be greatly appeciated,  Thanks in advance.

![Снимок экрана (297)](https://github.com/Cyfrin/foundry-full-course-f23/assets/112232336/1f84f237-682f-4043-ad69-696e1ee98f8c)

**Comment by Abelaby** [comment]
Hey @ZanyBonzy the following command worked for me in Windows running Ubuntu with WSL.

```
sudo apt install make        # version 4.3-4.1build1

```

This was recommended by my terminal itself and I just copied and pasted it, you will be prompted for your password.
After installation run 

```
make
```

Hope this helps

**Reply by shikhar229169** [reply]
'S' in sudo should be in lowercase

**Reply by Abdulllah-Rizwan** [reply]
Thanks a ton @shikhar229169  it worked for me easily

**Reply by zubyoha** [reply]
Worked for me too!

**Reply by YusufsDesigns** [reply]
Same here

**Comment by Renzo1** [comment]
I had the same issue. This helped me solve it.

https://stackoverflow.com/questions/32127524/how-to-install-and-use-make-in-windows

**Reply by greaterheightscomputer** [reply]
sudo apt install make        # version 4.3-4.1build1
please what will the password be?

**Reply by Keivou** [reply]
The guide with Chocolatey worked brilliantly, it really helps to read what the cli tells you ;)

**Reply by YavorJJ** [reply]
does Chocolately work when you already have ubuntu installed? thanks for your reply!

**Comment by greaterheightscomputer** [comment]
Appreciate @ZanyBonzy  I add my personal mail password and it works. Thanks in millions

**Reply by YavorJJ** [reply]
what happned with your password how did you fix it? ubuntu wants my password but I forgot it.
